.github/workflows/verification.yml

```diff
@@ -0,0 +1,23 @@
+name: Static Checks
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
```

Comment:
What is the reason to chose this particular version?

Reply:
3.9 is the most recent release version of Python, and now the most recent version supported by COBalD.

Reply:
So this is version specific?

Reply:
In a way, yes – some future version of COBalD might not run on 3.9 anymore. As of now, 3.9 will be adequate for a while though ([EOL is 2025](https://www.python.org/dev/peps/pep-0596/)).

Reply:
Are you aware of a way to couple this to the version information´, so it would get automatically changed when it is updated?

Reply:
I'm not aware of any way to automatise this.

Reply:
Okay. Maybe this is something what would be nice to have, but not necessary.